KarhooUISDK/CommonUI/VehicleCapacityView/NewVehicleCapacityView.swift

```diff
@@ -0,0 +1,157 @@
+//
+// Created by Bartlomiej Sopala on 10/03/2022.
+// Copyright (c) 2022 Flit Technologies Ltd. All rights reserved.
+//
+
+import Foundation
+import UIKit
+import KarhooSDK
+
+public struct KHNewVehicleCapacityViewID {
+    public static let capacityView = "vehicle_capacity_view"
+    public static let passengerContainerView = "passenger_stack_container"
+    public static let passengerStack = "passenger_stack"
+    public static let baggageContainerView = "baggage_stack_container"
+    public static let baggageStack = "baggage_stack"
+    public static let baggageIcon = "baggage_image"
+    public static let baggageCapacityLabel = "baggage_capacity_label"
+    public static let capacityIcon = "passenger_capacity_image"
+    public static let passengerCapacityLabel = "passenger_capacity_label"
+}
+
+final class NewVehicleCapacityView: UIStackView {
+
+    private lazy var passengerContainerView = UIView().then { view in
+        view.backgroundColor =  KarhooUI.colors.background1
+        view.accessibilityIdentifier = KHNewVehicleCapacityViewID.passengerContainerView
+        view.clipsToBounds = true
+        view.layer.cornerRadius = UIConstants.CornerRadius.xSmall
+    }
+
+    private lazy var passengerStack = UIStackView().then {stack in
+        stack.accessibilityIdentifier = KHNewVehicleCapacityViewID.passengerStack
+        stack.spacing = UIConstants.Spacing.xxSmall
+    }
+
+    private lazy var baggageContainerView = UIView().then { view in
```

Comment:
We could create a generic content view that contains the stack, image view and label. Then customize and add as many of them as we want into the NewVehicleCapacityView. My reasoning is that here we only need passenger and baggage, but maybe in the checkout screen we'll want more info shown in this format, or maybe in the trip details. It would make it more flexible basically, not limited to just these 2 pieces of information.

Reply:
done